src/pal/tools/gen-buildsys-clang.sh

```diff
@@ -40,7 +42,11 @@ else
   exit 1
 fi
 
-desired_llvm_version=3.5
+if [ $OS = "FreeBSD" ]; then
+	desired_llvm_version=35
+else
+	desired_llvm_version=3.5
+fi
```

Comment:
@janhenke  could you please fix this one as well? (And also make the indentation to match the surrounding code.)